package/MDAnalysis/topology/PDBParser.py

```diff
@@ -39,8 +39,10 @@
 
 .. Note::
 
-   The parser processes atoms and their names. Masses are guessed and set to 0
-   if unknown. Partial charges are not set. Elements are parsed if they are
+   The parser processes atoms and their names.
+   Partial charges are not set. Elements are parsed if they are
+
+
```

Comment:
```suggestion
   Partial charges are not set. Elements are parsed if they are
```
Just removing some extra empty lines!